Update edge-webview-interactive-dragging flag (#3183)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/webview-features-flags.md

| `edge-webview-foreground-boost-opt-in` | Opts-in to foreground boost. |
| `edge-webview-foreground-boost-opt-out` | Opts-out of foreground boost. |
| `edge-webview-force-personal-context` | Forces Edge WebView browser processes to run in WIP personal context. |

> | `edge-webview-interactive-dragging` | Enables pointer events and focus events to occur on elements that have the `--app-region: drag` attribute.  If this flag isn't set, such elements aren't interactive. |

Reviewer: ```suggestion
| `edge-webview-interactive-dragging` | Enables pointer events and focus events to occur on elements that have the `--app-region: drag` attribute.  If this flag isn't set, drag elements are non-interactive. |
```
